### PR TITLE
Sneakers

### PR DESCRIPTION
**Pull request by Kmc2000**
### Intent of your Pull Request

Adds some new clothing items.
Included are a few shoes, some sweatshirts, as well as clothing cleaning sprays that allow you to get the blood out of your new stuff.

(This time they aren't as big as people)

#### Changelog

:cl: Kmc2000
rscadd: Added some new clothes, look under the miscellaneous tab in cargo.
rscadd: Added cleaning sprays for clothing, these can be bought by cargo.
/:cl:

### CODE REVIEW

**Comment by fluffe9911** [comment]
Oh sweet cant wait to wear dem sweet kicks lol

**Comment by optimumtact** [comment]
DAMN DANIEL

**Comment by Smudged** [comment]
Well, more shoes to stick in the fire

**Comment by Kmc2000** [comment]
I'm coming for you smudgels

**Comment by fluffe9911** [comment]
Begone orange you have no power here in this realm!!!

Edit: reeeeeeee

File: code/modules/cargo/yog_packs.dm [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/datum/supply_pack/misc/miscclothing
```
Comment (Amelia0010) [inline code/modules/cargo/yog_packs.dm]: this whole file should be yogstation/code/modules/cargo/cargo_packs.dm

File: yogstation/code/modules/clothing/shoes/special_shoes.dm [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	to_chat(user, "This bottle has [charges*10] ml of [max_charges*10] ml left in it")
+	return
+
+/obj/item/shoe_protector/afterattack(obj/I, mob/user)
+	if(infinite && charges <=0)
+		to_chat(user, "WARNING! Low liquid volume detected. Engaging time-jump.")
+		do_time_jump()
+		return 0
+	if(istype(I, /obj/item/clothing))
+		if(charges > 0)
```
Comment (Amelia0010) [inline yogstation/code/modules/clothing/shoes/special_shoes.dm]: just do if(charges)

File: yogstation/code/modules/clothing/shoes/special_shoes.dm [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+/obj/item/shoe_protector/examine(mob/user)
+	. = ..()
+	to_chat(user, "This bottle has [charges*10] ml of [max_charges*10] ml left in it")
+	return
+
+/obj/item/shoe_protector/afterattack(obj/I, mob/user)
+	if(infinite && charges <=0)
+		to_chat(user, "WARNING! Low liquid volume detected. Engaging time-jump.")
+		do_time_jump()
+		return 0
```
Comment (Amelia0010) [inline yogstation/code/modules/clothing/shoes/special_shoes.dm]: return FALSE

File: yogstation/code/modules/clothing/shoes/special_shoes.dm [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+	addtimer(CALLBACK(src, .proc/do_time_jump_return), 50)
+
+/obj/item/shoe_protector/proc/do_time_jump_return()
+	playsound(src.loc, 'sound/effects/empulse.ogg', 100, 1)
+	playsound(get_turf(src), 'sound/magic/ethereal_enter.ogg', 50, 1, -1)
+	icon_state = initial(icon_state)
+	charges = max_charges
+	infinite = TRUE
+	SpinAnimation(0,0)
+	visible_message("[src] pops back into existence! it's fully refilled.")
```
Comment (Amelia0010) [inline yogstation/code/modules/clothing/shoes/special_shoes.dm]: capital I in It's

File: yogstation.dme [hunk 5]
```diff
@@ -1354,6 +1354,7 @@
 #include "code\modules\cargo\order.dm"
 #include "code\modules\cargo\packs.dm"
 #include "code\modules\cargo\supplypod_beacon.dm"
+#include "code\modules\cargo\yog_packs.dm"
```
Comment (Amelia0010) [inline yogstation.dme]: 🤔

**Pull request merged**